pkg/cmd/server/origin/normalize_policy_hook.go

```diff
@@ -0,0 +1,90 @@
+package origin
+
+import (
+	"strings"
+
+	"github.com/openshift/origin/pkg/authorization/api"
+	authorizationinternalversion "github.com/openshift/origin/pkg/authorization/generated/internalclientset/typed/authorization/internalversion"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/apimachinery/pkg/util/sets"
+	genericapiserver "k8s.io/apiserver/pkg/server"
+	coreclient "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/typed/core/internalversion"
+)
+
+func NormalizePolicyPostStartHook(hookContext genericapiserver.PostStartHookContext) error {
+	authorizationClient, err := authorizationinternalversion.NewForConfig(hookContext.LoopbackClientConfig)
+	if err != nil {
+		utilruntime.HandleError(err)
+		return nil
+	}
+	if err := normalizeClusterPolicy(authorizationClient.ClusterPolicies()); err != nil {
+		utilruntime.HandleError(err)
+	}
+	coreClient, err := coreclient.NewForConfig(hookContext.LoopbackClientConfig)
+	if err != nil {
+		utilruntime.HandleError(err)
+		return nil
+	}
+	namespaces, err := coreClient.Namespaces().List(v1.ListOptions{})
+	if err != nil {
+		utilruntime.HandleError(err)
+		return nil
+	}
+	for _, namespace := range namespaces.Items {
```

Comment:
post start hook is nice, but thinking about it, this is going to take a looong time.  I see two possibilities:
 1. do it in the controller we have and check in the migration check.
 2. do in oadm migrate. 

@enj Which would you do?